org.adempiere.base/src/org/compiere/util/CLogFile.java

```diff
@@ -334,6 +333,19 @@ public synchronized void publish (LogRecord record)
 		}
 	}	//	publish
 
+	private String getPrefix()
+	{
+		try
+		{
+			return Env.parseContext(Env.getCtx(), 0, System.getProperty("org.idempiere.FileLogPrefix"), false);
+		}
+		catch (Exception ex)
+		{
+			reportError ("LogFilePrefix", ex, ErrorManager.FORMAT_FAILURE);

```

Comment:
perhaps we should consider to set s_fileLogPrefix to empty here to avoid repeating the parse exception for every logging call.